README.md

```diff
@@ -375,31 +363,35 @@ Please execute at least step 1 + 2 if before importing restdocs-wiremock into yo
 
 Given that the `master` branch on the upstream repository is in the state from which you want to create a release, execute the following steps:
 
-```shell
-git checkout master
-git pull upstream master
+1. Create release via the GitHub UI
 
-# Print current version to the terminal
-./gradlew currentVersion
+https://github.com/ePages-de/restdocs-wiremock/releases/new
 
-# (a) Release new patch version
-./gradlew release -Prelease.versionIncrementer=incrementPatch
+Use the intended version number as "Tag version", e.g. "0.8.0".
 
-# (b) Release new minor version
-./gradlew release -Prelease.versionIncrementer=incrementMajor
+This will automatically trigger a Travis build which publishes the JAR files for this release to Sonatype.
 
-## (c) Release new major version
-./gradlew release -Prelease.versionIncrementer=incrementMajor
+2. Login to Sonatype
 
-# Print current version to the terminal
-./gradlew currentVersion
-```
+Login to Sonatype and navigate to the [staging repositories](https://oss.sonatype.org/#stagingRepositories).
+
+3. Close the staging repository
+
+Select the generated staging repository and close it.
+Check that there are no errors afterwards (e.g. missing signatures or Javadoc JARs).
+
+4. Release the repository
+
+Select the genereted staging repository and publish it.
+Soon after, the release should be available in the in the "Public Repositories" of ePages:
```

Comment:
Small typos

```suggestion
Select the generated staging repository and publish it.
Soon after, the release should be available in the "Public Repositories" of ePages:
```

Reply:
Here again, I would inline the link in the text, i.e.:
```suggestion
Select the genereted staging repository and publish it.
Soon after, the release should be available in the ["Public Repositories" of ePages](https://oss.sonatype.org/service/local/repo_groups/public/content/com/epages/restdocs-wiremock/maven-metadata.xml).
```